### ISSUE TITLE
missing config

### ISSUE THREAD

**Issue by joewilliams** [ongardie/raft.tla#1]
Can you include the TLA+ config you use in this repo?

**Comment by ongardie** [comment]
I've forgotten what that even means, to be honest. The spec isn't meant to be run in any way.

I have an old config file around there, though it looks like most everything is commented out.
```
CONSTANTS Server = {r1,r2,r3}
          Value = {v1,v2}
          Follower = Follower
          Candidate = Candidate
          Leader = Leader
          Nil = Nil
          RequestVoteRequest = RequestVoteRequest
          RequestVoteResponse = RequestVoteResponse
          AppendEntriesRequest = AppendEntriesRequest
          AppendEntriesResponse = AppendEntriesResponse
          TLC_MAX_TERM = 3
          TLC_MAX_ENTRY = 1
          TLC_MAX_MESSAGE = 1
\*          PNat = {1,2,3,4,5}
\*          Nat = {0,1,2,3,4,5}
\*SYMMETRY Perms
SPECIFICATION Spec
\*CONSTRAINT TermConstraint
\*CONSTRAINT LogConstraint
\*CONSTRAINT MessageConstraint
\*INVARIANT AtMostOneLeaderPerTerm
\*INVARIANT TermAndIndexDeterminesLogPrefix
\*INVARIANT StateMachineSafety
\*INVARIANT NewLeaderHasCompleteLog
\*INVARIANT CommitInOrder

\*INVARIANT MessageTypeInv
\*INVARIANT TypeInv
```

**Comment by joewilliams** [comment]
Cool, thanks that looks right.

**Comment by ongardie** [comment]
I could seriously use a reminder here, is the config file something you need for TLC, or do other tools need it?

**Comment by joewilliams** [comment]
@ongardie oh sorry, yes, TLC needs a config otherwise it doesn't know what the CONSTANTS are in the model.

**Comment by ongardie** [comment]
@joewilliams gotcha, yeah. You'll probably need to tweak the spec before you're able to run with TLC again. I found that if things got too mathy (\exists, for example), they'd slow down TLC severely.

**Comment by deardeng** [comment]
![image](https://user-images.githubusercontent.com/3887565/159827790-35733f02-6721-4cab-9683-89ea2fed40ac.png)
![image](https://user-images.githubusercontent.com/3887565/159827830-41a97fb3-8f21-491a-a46f-496e30e211be.png)
hi, I use this, but it used 567G disk capacity, and finally exhausted the disk and stopped, and it didn't finish.

> ongardie

**Comment by ongardie** [comment]
@ukd1 if you get a config file to run and submit a PR, I'd be happy to merge it. I just don't want to make non-trivial changes to the core spec at this point.

**Comment by ongardie** [comment]
> Also, do you recalll how long the model took to run with this config?

No, and I think that's the issue. I think the spec probably needs changes to be practically runnable. @deardeng reported running out of space above, for example.

**Comment by ongardie** [comment]
I've reopened this issue so it's a little more discoverable.
